setup.py

```diff
@@ -27,8 +27,10 @@
 http://docs.python.org/2/distutils/sourcedist.html
 
 """
-import os
 from setuptools import setup, find_packages
+from sys import version_info
+import os
```

Comment:
No, I did not say that separating stdlib/3rd party/local imports "looked weird". Why do you think I added the blank line on line 33? I thought your issue was with the ordering _within_ those blocks?

FWIW, PEP8 doesn't specify an order within those groups, it doesn't even bother with alphabetical order in some of its examples, see the section about importing classes from a module as an example.